src/Console/Command/CheckCommand.php

```diff
@@ -40,6 +41,7 @@ protected function configure()
                     ),
                     new InputOption('no-cache', null, InputOption::VALUE_NONE, 'Disable rule set cache'),
                     new InputOption('cache-dir', null, InputOption::VALUE_REQUIRED, 'Cache directory', '.rules/'),
+                    new InputOption('log-html', null, InputOption::VALUE_OPTIONAL, 'Log output in HTML format to file.'),
```

Comment:
This should be `InputOption::VALUE_REQUIRED` or the code should be changed to use a default location for the HTML file.